Description: Use the external sigar library.
 Use the external sigar library.
--- a/bin/elasticsearch.in.sh
+++ b/bin/elasticsearch.in.sh
@@ -1,6 +1,7 @@
 #!/bin/sh
 
-ES_CLASSPATH=$ES_CLASSPATH:$ES_HOME/lib/${project.build.finalName}.jar:$ES_HOME/lib/*:$ES_HOME/lib/sigar/*
+ES_CLASSPATH=$ES_CLASSPATH:$ES_HOME/lib/${project.build.finalName}.jar:$ES_HOME/lib/*:/usr/share/java/sigar.jar
+JAVA_OPTS="$JAVA_OPTS -Djava.library.path=/usr/lib/$(arch)-linux-gnu/jni"
 
 if [ "x$ES_MIN_MEM" = "x" ]; then
     ES_MIN_MEM=256m
